frontend/src/pages/CreateUser.tsx

```diff
@@ -1,17 +1,8 @@
-import { FormEvent, useState, useContext } from "react"
+import { FormEvent, useState } from "react"
 import { useNavigate } from "react-router-dom"
 import { registerUser } from "../api/backendClient"
-import { UserSignUpContext } from "../Components/UserManager/UserSignUpManager"
 
 export function CreateUser(): JSX.Element {
-  const {
-    saveUsernameToContext,
-    savePasswordToContext,
-    saveEmailToContext,
-    saveFirstnameToContext,
-    saveLastnameToContext,
-    saveAboutmeToContext,
-  } = useContext(UserSignUpContext)
```

Comment:
a general note for reviews, I had asked about this in my previous review of this branch, and now it's been removed - I'm going to infer that you guys decided it shouldn't be used for some reason.

However, it would be good practice to write a message explain what this is/why it's been removed or something to give the reviewer context about why you've made the change, that helps them stay informed about what's going on and why